configure.ac

```diff
@@ -697,9 +697,10 @@ AS_IF([test x"$enable_intrinsics" = x"yes"],[
              __m256i mtest2;
              mtest = _mm256_set1_ps((float)time(NULL));
              mtest = _mm256_fmadd_ps(mtest, mtest, mtest);
-             mtest1 = _mm256_set_m128i(_mm_loadu_si64(utest), _mm_loadu_si64(utest));
+             mtest1 = _mm256_set_m128i(
+              _mm_loadl_epi64((__m128i_u *)utest), _mm_loadl_epi64((__m128i_u *)utest));
```

Comment:
As far as I can tell, __m128i_u is just an internal gcc/clang thing and not something we can rely on unfortunately.

Reply:
Solution?